.ci/jenkins_job_build.sh

```diff
@@ -60,6 +61,15 @@ then
 	git fetch origin "pull/${pr_number}/head:${pr_branch}"
 	git checkout "${pr_branch}"
 	git rebase "origin/${ghprbTargetBranch}"
+
+	# As we currently have CC and runv runtimes as git submodules
+	# we need to update the submodules in order to get
+	# the correct changes.
+	# This condition should be removed when we have one runtime.
+	if [ "${kata_repo}" == "${runtime_repo}" ]
```

Comment:
we possibly could have just done this unconditionally, but checking for the known repo is fine :-)

Reply:
Yeah, it's really a nit since the submodules are temporary anyway, but you could do the following:

```
[ -f .gitmodules ] && git submodule update
```

Reply:
I like it, fixed.